src/data/selector.ts

```diff
@@ -124,6 +125,13 @@ export interface ConstantSelector {
   } | null;
 }
 
+export interface CountrySelector {
+  country: {
+    multiple?: boolean;
+    countries: string[];
```

Comment:
These are not implemented right now?

Reply:
No, still thinking on the best approach to move forward (this is not my strong suit).
- Needs to filter
- Needs to allow selecting multiple.

So probably this would become very similar to `SelectSelector`

Reply:
Not perse, for multiple you could just have multiple country selectors, like we do for the entity selector for example.

Are there already usecases for multiple? Or could we start with not supporting multiple?

For the `countries` option you want to look at the language selector and picker, that is very similar.

If you need help, feel free to ping me on discord.

Reply:
Thanks @bramkragten 
I removed multiple for now. Probably as you say no immediate needs.
It's now functional and looks like you mentioned very similar to language.

However if I understand it correctly this does only work with English right now so should we consider to add the users language into this?